sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -15,6 +13,7 @@ import {
   isTokenCredential
 } from "@azure/core-amqp";
 import { SubscriptionClient } from "./subscriptionClient";
+import { WebSocketOptions } from "@azure/core-amqp";
```

Comment:
Same comment here about updating the package.json to set the minimum version of core-amqp to what's in master.

Reply:
Done.